test/string_util_gtest.cc

```diff
@@ -215,4 +216,4 @@ TEST_P(HumanReadableFixture, HumanReadableNumber) {
   ASSERT_THAT(str, ::testing::MatchesRegex(std::get<2>(GetParam())));
 }
 
-}  // end namespace
+}  // end namespace
```

Comment:
Please add newline back

Reply:
I did noticed it but thought maybe clang-format did that. Doing it.

Reply:
I think this went into a new line as I added one more test case at line 173 in this file `std::make_tuple(1234 * 1000, "1.234M"),`

Reply:
Not done.